cmd/vl3-nse/aws_tgw_connect.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"context"
+	"github.com/cisco-app-networking/nsm-nse/pkg/nseconfig"
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/connection"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/networkservice"
+	"github.com/networkservicemesh/networkservicemesh/sdk/endpoint"
+	"github.com/sirupsen/logrus"
```

Comment:
Imports should be placed in 3 groups:

- Standard library
- Third-party packages
- First-party packages from outside repositories (e.g. github.com/org/path/to/import)